docs/en_us/platform_api/source/courses/blocks.rst

```diff
@@ -0,0 +1,350 @@
+.. _Courses API Blocks Resource:
+
+########################################
+Courses API Blocks Resource
+########################################
+
+With the Courses API **Blocks** resource, you can complete the
+following tasks.
+
+
+.. contents::
+   :local:
+   :depth: 1
+
+.. _Get a list of the course blocks in a course:
+
+*******************************************************
+Get a List of Course Blocks in a Specified Course
+*******************************************************
+
+The endpoint to get a list of course blocks in a specified block tree
+``/api/courses/v1/blocks/?course_id=<course_id>``.
+
+=====================
+Use Case
+=====================
+
+Get a list of course blocks in a specified course. Response results depend on
+the requesting user's access level.
+
+=====================
+Example Request
+=====================
+
+GET /api/courses/v1/blocks/?course_id=<course_id>
+
+GET /api/courses/v1/blocks/?course_id=edX%2FDemoX%2FDemo_Course
+&all_blocks=true&requested_fields=graded,format,student_view_multi_device
+
+=====================
+Parameters
+=====================
+
+This view redirects to /api/courses/v1/blocks/<root_usage_key>/ for the root
+block usage key of the course specified by course_id.
```

Comment:
I don't understand this paragraph. Why mention the view? This is the only reference to "root_usage_key." Can we just remove this paragraph?